src/Microsoft.OData.Edm/Vocabularies/CapabilitiesVocabularies.xml

```diff
@@ -11,14 +11,10 @@
   </EnumType>
 
   <!-- Request Capabilities -->
-
-  <!-- Model validation failed as "The value of the property 'IEdmValueAnnotation.Value' must not be null."
-
   <Term Name="SupportedFormats" Type="Collection(Edm.String)">
     <Annotation Term="Core.Description" String="Media types of supported formats, including format parameters" />
-    <Annotation Term="Core.IsMediaType" />
+    <Annotation Term="Core.IsMediaType" Bool="true" />

```

Comment:
The file should be the same to: http://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/vocabularies/Org.OData.Capabilities.V1.xml
```xml
<Annotation Term="Core.IsMediaType"/>
```